lib/zendesk_apps_support/validations/translations.rb

```diff
@@ -7,6 +7,7 @@ module Validations
     module Translations
       TRANSLATIONS_PATH = %r{^translations/(.*)\.json$}
       VALID_LOCALE      = /^[a-z]{2}(-\w{2,3})?$/
+      MANDATORY_KEYS = %w(name description installation_instructions long_description).freeze
```

Comment:
Sean, I think we are using short_description as mandatory in apps_approval ( and accepting description if present )